content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -45,7 +45,7 @@ restarts. Here is the configuration file for the Pod:
 
    The output looks like this:
 
-   ```shell
+   ```console
```

Comment:
so the changes from `shell` to `console` is for output only?

Reply:
Yes, now it is. I checked repo, we have three variants:

- `console`
- `none`
- keep empty

`shell` is applicable to a command like `kubectl get`

Reply:
I agree that `shell` is more applicable than `console` and to keep the original